tactic/scc.lean

```diff
@@ -0,0 +1,167 @@
+/-
+Copyright (c) 2018 Simon Hudon All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Simon Hudon
+
+Data structures and algorithms for finding the sets of equivalent
+proposition in a set of implications.
+-/
```

Comment:
What does `scc` stands for?

Reply:
strongly connected components in graph theory. I can add that to the header.